src/lib/setup.js

```diff
@@ -213,6 +219,14 @@ export default function(app) {
   app.use(unsupportedBrowserCheck)
   app.use(splitTestMiddleware)
 
+  if (DD_API_KEY && DD_APM_ENABLED) {
+    // DataDog automatically hooks into Express
+    // https://datadog.github.io/dd-trace-js/index.html
+    // and we use all the defaults from:
+    // https://github.com/DataDog/dd-trace-js/blob/master/docs/API.md#tracer-settings
+    ddTracer.init({})
```

Comment:
I don't think the running app will have a datadog API key (since that's the concern of the trace agent running elsewhere). The `..._ENABLED` one might be applicable though, assuming we'll set this in staging/prod config-maps.